### ISSUE TITLE
`warn` behavior

### ISSUE THREAD

**Issue by eouia** [baryon/tracer#116]
I love your module, but I have a question. 

In Chromium dev console, `tracer.warn` is displayed as `error`. Of course, under standards, it might be right to be displayed as `error`. But this might give some sign of fear to users. I'm hesitating to use `warn`. Could it be changed like Chromium devTools?   
![image](https://user-images.githubusercontent.com/1720610/81834658-88472c00-9541-11ea-96a7-70b010c9c469.png)

**Comment by eouia** [comment]
Solved.
```js
  transport: function(data) {
    if (data.level == 4) {
      console.warn(data.output)
    } else if (data.level > 4) {
      console.error(data.output)
    } else (
      console.log(data.output)
    )
  },
```

**Comment by baryon** [comment]
Fixed on v1.1.0
